languages/python/tree-sitter-python.cabal

```diff
@@ -35,6 +35,18 @@ library tree-sitter-python-internal
   c-sources:           vendor/tree-sitter-python/src/parser.c
                      , vendor/tree-sitter-python/src/scanner.cc
   extra-libraries:     stdc++
+  ghc-options:         -Weverything -Wno-missing-local-signatures -Wno-missing-import-lists -Wno-implicit-prelude -Wno-safe -Wno-unsafe -Wno-name-shadowing -Wno-monomorphism-restriction -Wno-missed-specialisations -Wno-all-missed-specialisations -Wno-missing-export-lists -ddump-splices
```

Comment:
Kill the `-ddump-splices` on this line here, too.

Reply:
Good catch, thank you!